### PR TITLE
Initial Styles for Ember Theming

### PR DESCRIPTION
**Pull request by BrianSipple**
I know our UI is still VERY early-days — and of course any _real_ design is something that deserves a good amount of consideration at some point — but I couldn’t help myself from playing around with our styles and setting up some theming to match the Ember [Guides](https://guides.emberjs.com/v2.5.0/) and [Website](http://emberjs.com/).

screen shot 2016-05-31 at 12 25 27 pm

Like I said, still a bunch more that could be polished -- but I figured this might be a decent first pass. 😄

### CODE REVIEW

**Comment by binhums** [comment]
Oh nice work! This is a big improvement.

It would be nice to eventually have consistency between our ember-a11y projects. @BrianSipple have you seen some of @MelSumner's work on https://ember-a11y.github.io/a11y-demo-app/.

**Comment by Robdel12** [comment]
I'm curious to see if these colors pass the contrast tests in `ember-a11y-testing`? I remember pairing with @toddjordan on GAAD on an app that had a similar color scheme which failed the tests.

**Comment by nathanhammond** [comment]
I've written _so_ little CSS recently. In fact, [what you're replacing](https://github.com/ember-a11y/ember-a11y/blob/master/tests/dummy/app/styles/app.scss), may be the extent of the CSS that I've written this year.

Being that this is some combination of a demo application and a testing application all rolled into one, I'm in favor of making it look good. 👍

File: tests/dummy/app/styles/app.scss [hunk 1]
```diff
@@ -1,8 +1,96 @@
...
+}
+
+h1, h2, h3, h4, h5, h6 {
+  color: $theme-color__primary;
+  margin: 0;
+  line-height: 2.25;
+}
 
 .focusing-outlet:focus {
   outline: yellow auto 5px;
```
Comment (nathanhammond) [inline tests/dummy/app/styles/app.scss]: Can we make sure that this is still enough contrast?

**Comment by BrianSipple** [comment]
@Robdel12 @nathanhammond You both are right on with the potential contrast issue. 

Running an accessibility audit in DevTools sees warning crop up for all of the colored headings and white-on-orange anchor text. Furthermore, running the hex values for our primary orange through [WebAIM's color contrast checker](http://webaim.org/resources/contrastchecker/) failed with some rather mediocre scores:

screen shot 2016-06-01 at 3 41 38 am
screen shot 2016-06-01 at 4 04 47 am
screen shot 2016-06-01 at 4 28 42 am

So... I think we've discovered a really cool challenge that I wasn't considering initially, but could readily solve on this branch: 

We could 1) try to find a palette that still resembles Ember’s branding colors but DOES pass contrast tests, or 2) create a different palette entirely to give the Ember A11y Project its own unique look and feel (while also passing contrast tests, of course). 

Regarding Option 2, the site for [Ember Fastboot](https://www.ember-fastboot.com/) is a perfect example of this, as is what we have at https://ember-a11y.github.io/a11y-demo-app/ -- pretty spiffy stuff, @MelSumner.

**Comment by nathanhammond** [comment]
@BrianSipple I like an extension of option one, let's work with the learning team (hi @acorncom, @locks, @toddjordan, and @wifelette!) to upstream an accessibility-aware color scheme.

Notes:
* Existing palette from the logos:
  * `#E34C32` (orange)
  * `#612116` (brown)
  * `#FFFAF1` (off-white) [(the ember-data-on-dark logo uses pure white, mistake?)](http://emberjs.com/images/brand/ember_Data-Dark.png)
* We may want to work with @in (the original designer of the Ember.js site) or whoever has been responsible for that recently.

All: does that sound reasonable?

**Comment by locks** [comment]
The Guides, which seems to be the basis for OP's colour scheme, were elaborated by the ThoughtBot people, with some tweaks by us to increase the legibility. As mentioned above, the palette from the logos can't really be altered, but there shouldn't be a problem tweaking the website colours.

The [styleguide](https://github.com/locks/ember-styleguide) we started extracting from Ember resources was to both clarify the styling of Ember assets, as well as to make it easy to share and keep the assets up to date.

**Comment by nathanhammond** [comment]
@MelSumner We need a color stack that is going to meet the needs of both form and function and (hopefully) apply across many Ember web properties. Presumably, the only colors we can't change are listed above, but we should probably aim to darken/lighten existing theme colors so that we maintain branding.

Unfortunately, [we didn't factor out all color references into variables](https://github.com/emberjs/website/tree/master/source/stylesheets) for the website, so it's hard for us to grab a collection of colors and just redefine them. [A recent pass for creating an ember styleguide is closer and probably where we want to start from.](https://github.com/locks/ember-styleguide/blob/master/app/styles/base/_variables.scss)

[All I know how to do is this,](https://color.adobe.com/create/color-wheel/?base=0&rule=Custom&selected=0&name=Ember%20Theme&mode=rgb&rgbvalues=0.8901960784313725,0.2980392156862745,0.19607843137254902,0.3803921568627451,0.12941176470588237,0.08627450980392157,1,0.9803921568627451,0.9450980392156862,0.39019607843137255,0.23050207673599019,0.2030050963116524,0.6901960784313725,0.23107886326328683,0.15202556793642563&swatchOrder=0,1,2,3,4) so anything you can come up with will be fantastic.

I'm going to create a meta tracking issues repo under ember-a11y for us to keep up with a11y issues across multiple Ember community projects.

**Comment by BrianSipple** [comment]
@nathanhammond http://colorsafe.co/ is a really handy tool for creating accessible color palettes. 

Also, now that we're [tracking this more broadly](https://github.com/ember-a11y/issues/issues/1) I'm wondering if the changes here couldn't serve as a temporary improvement (someone once referred to it as a "big" improvement over the current implementation 😉).

I updated the orange to #E34C32 and placed it against an offWhite (#FFFAF1) background -- which has a fairly solid 3.78 contrast ratio.

**Comment by nathanhammond** [comment]
@BrianSipple As the Ember A11y team I think we need to set the example of making sure we meet all constraints before landing changes. It is a big improvement for sighted users, and a step back for low sight users. Figure out some colors that get us to WCAG AA and I'm in.

**Comment by MelSumner** [comment]
@BrianSipple @nathanhammond : the section titles will pass if the font size is increased, and if the text is kept at that brown color (please no, let's talk about that!) then it needs to be at least 14px. 

Now, I am of the (16px minimum no matter what srsly what's with all the tiny type) camp, but if that's not where everyone is, then we definitely need to darken the text colors, at the least. 

Minimum viable change: We can increment in darker  steps (maybe 2% increments) until it passes. This will give it the same feel without drastically altering the look (as much as I love the way ember-a11y demo site design turned out!). 

It also may assist (from a user psychology standpoint) in adoption if we don't change it too drastically. It's easier for the human mind to accept a shift in a specific/different direction if it doesn't send up huge red flags of "hey! this is way different!!"

So, to sum up, here are our options, from least change to most change:
1. darken the colors of all text on the #FFFBF5 background
2. both darken the text colors and increase the font size
3. darken the text colors and increase the font size for now, but make it look and feel like the ember-a11y demo site to keep the feeling consistent across the same theme of project(s). 

Thoughts?

**Comment by MelSumner** [comment]
Also, here are two potential color schemes: 
https://color.adobe.com/ember-a11y-color-theme-8152188/
https://color.adobe.com/ember-a11y-option-2-color-theme-8152558/

**Comment by Robdel12** [comment]
I like option 2 more, but both are really good.

**Comment by BrianSipple** [comment]
@MelSumner These are both really nice! 

@ember-a11y/core, if we're in agreement that the color palette here can diverge a bit, at least temporarily, from the more [standard community palette that we're looking for](https://github.com/ember-a11y/issues/issues/1), I'd love to get something together around @MelSumner's ideas on this branch -- and then focus on long-term design.

**Comment by nathanhammond** [comment]
I'm going to bow out of this one–it's in far better hands as long as it's not in mine. 😛 I'm generally of the opinion that just choosing something and going with it is better than talking about it for too long. In that vein I don't care in the slightest about the divergence, it's likely a consequence of not giving ourselves enough room to grow in the initial palette and will show up in that rework too.

**Comment by BrianSipple** [comment]
I took the colors from @MelSumner's [second palette](https://color.adobe.com/ember-a11y-option-2-color-theme-8152558/) and fleshed them out into different variables, and then got to playing around with applying them on the page. 

On the whole, everything is still a bit experimental, but I was able to get the number of audit contrast warnings down to 0:

screen shot 2016-06-04 at 4 15 06 pm


I also tweaked the font to a near-black (#1C1C1D) so that it contrasted well with the off-white background.
